### PR TITLE
Tested the jwt cli

### PR DESCRIPTION
**Pull request by mohammadsheikhian**
**Please checked this pull request after https://github.com/Carrene/restfulpy/pull/312**

### CODE REVIEW

File: restfulpy/tests/test_jwt_cli.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import json
```
Comment (ArashFatahzade) [inline restfulpy/tests/test_jwt_cli.py]: Please use `ujson` module instead of `json`.

File: restfulpy/tests/test_jwt_cli.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import pytest
+from bddcli import Given, given, when, stdout, stderr, Application
+from itsdangerous import TimedJSONWebSignatureSerializer
+from itsdangerous.exc import SignatureExpired
+from nanohttp import settings
+
+from restfulpy import Application as RestfulPyApplication
```
Comment (ArashFatahzade) [inline restfulpy/tests/test_jwt_cli.py]: I think `restfulpy` is one word. `RestfulpyApplication` is better than `RestfulPyApplication`.

File: restfulpy/tests/test_jwt_cli.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        assert pirincipal.loads(token) == {}
+
+        # Create a jwt token with a payload
+        payload = dict(a=1)
+        when(given + f'\'{json.dumps(payload)}\'')
+        assert stderr == ''
+        token = f'{stdout}'[:-1]
+        assert pirincipal.loads(token) == payload
+
+        # Create a expire token
```
Comment (ArashFatahzade) [inline restfulpy/tests/test_jwt_cli.py]: `expired`

**Comment by mohammadsheikhian** [comment]
Dear @pylover please check it out.

**Pull request merged**